src/app/map/Chat/_hooks/useAIChatIntegration.ts

```
@@ -0,0 +1,70 @@
+ * Hook that integrates AI chat functionality into the chat system
+ * This automatically sends user messages to the AI when they start with specific patterns
+ */
+export function useAIChatIntegration() {
+  const chatState = useChatState()
+  
+  // Try to use AI chat - might fail if cache context is not available
+  let sendToAI: ((message: string) => Promise<void>) | null = null
+  let isGenerating = false
+  
+  try {
+    const aiChat = useAIChat()
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)